local-api/formlabs-api-local-openapi.yaml

```diff
@@ -372,7 +383,7 @@ components:
               description: The total volume of models and supports in the scene
             unsupported_volume_ml:
```

Comment:
This name is also confusing... `volume_excluding_supports_ml`.

Reply:
+1